This patch is required to make "Start automatically on login" work

--- a/apps/desktop/src/main/messaging.main.ts
+++ b/apps/desktop/src/main/messaging.main.ts
@@ -136,7 +136,7 @@ export class MessagingMain {
   Version=${app.getVersion()}
   Name=Bitwarden
   Comment=Bitwarden startup script
-  Exec=${app.getPath("exe")}
+  Exec=bitwarden-desktop
   StartupNotify=false
   Terminal=false`;
 
